toxcore/group_chats.c

```diff
@@ -1495,9 +1503,16 @@ int group_packet_wrap(
         return -1;
     }
 
-    if (length > MAX_GC_PACKET_CHUNK_SIZE) {
-        LOGGER_ERROR(log, "Packet payload size (%u) exceeds maximum (%u)", length, MAX_GC_PACKET_CHUNK_SIZE);
-        return -1;
+    if (net_packet_type == NET_PACKET_GC_LOSSY) {
+        if (length > MAX_GC_CUSTOM_LOSSY_PACKET_SIZE) {
+            LOGGER_ERROR(log, "Packet payload size (%u) exceeds maximum (%u)", length, MAX_GC_CUSTOM_LOSSY_PACKET_SIZE);
+            return -1;
+        }
+    } else {
+        if (length > MAX_GC_PACKET_CHUNK_SIZE) {
+            LOGGER_ERROR(log, "Packet payload size (%u) exceeds maximum (%u)", length, MAX_GC_PACKET_CHUNK_SIZE);
```

Comment:
Avoid copy/pasting code. If you find yourself using copy and paste commands, think twice. At this point I think ?: isn't even the right answer, but instead make a function that returns the max size for a packet type.